OpenRA.Mods.Common/Activities/DeliverResources.cs

```diff
@@ -68,16 +76,19 @@ public override Activity Tick(Actor self)
 				foreach (var n in self.TraitsImplementing<INotifyHarvesterAction>())
 					n.MovingToRefinery(self, proc, null);
 
-				return ActivityUtils.SequenceActivities(self, movement.MoveTo(proc.Location + iao.DeliveryOffset, 0), this);
+				QueueChild(self, movement.MoveTo(proc.Location + iao.DeliveryOffset, 0), true);
+				return this;
 			}
 
 			if (!isDocking)
 			{
+				ChildActivity = new Wait(10);
```

Comment:
Seems a bit out of place to assign here directly, vs `QueueChild` in others?

Reply:
It is out of place. Fixed. 

Although I'm kind of wondering why it is here at all. It looks better without the pause. Was this behaviour in the original as well?